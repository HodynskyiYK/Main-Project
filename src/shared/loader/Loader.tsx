import React, { FC } from 'react'
import styles from './Loader.module.scss'

export const Loader: FC = () => {
    return (
        <div className={styles.loaderContent}>
            <span className={styles.loader}/>
        </div>
    )
}
