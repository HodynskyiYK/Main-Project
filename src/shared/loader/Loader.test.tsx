import React from 'react'
import { render } from '@testing-library/react'
import { Loader } from './Loader'

describe('Loader tests', () => {

    test('To match snapshot', () => {
        const {asFragment} = render(<Loader/>)
        expect(asFragment()).toMatchSnapshot()
    })
})