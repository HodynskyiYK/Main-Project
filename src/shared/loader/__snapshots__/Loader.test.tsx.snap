// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader tests To match snapshot 1`] = `
<DocumentFragment>
  <div
    class="loaderContent"
  >
    <span
      class="loader"
    />
  </div>
</DocumentFragment>
`;
